src/pages/AddProperty/AddProp.jsx

```diff
@@ -125,8 +126,8 @@ class AddProp extends React.Component {
                 <TopNav />
                 <div className="saveDraft-publish-buttons">
                     <button
-                        onClick={() => this.setState({draft: true})}>
-                        Save As Draft</button>
+                        onClick={this.submitProperty(true)}
+                    >Save As Draft</button>
                     <button id="pubButton">Publish Listing</button>
```

Comment:
Attach that same function to the publish button and set it to false. When you click the button does it work as expected? No errors in the console?

Reply:
Nothing happens. Although, when the page reloads, it gives me an alert to add more pictures.

Reply:
@ECruz49 I think you need to do `this.submitProperty.bind(true)` in order to pass the bool to the function or else the function invokes when the page is loaded which is why you are getting that alert. Try it out and maybe console.log the results in the function to see if it is working as expected